spec/factories/users.rb

```diff
@@ -1,13 +1,13 @@
 FactoryGirl.define do
   factory :user do
-    email "user@spec.com"
-    password "test1234"
+    email 'user@spec.com'
```

Comment:
I think email is uniqueness or should be so here you get an error when you create user twice. You should use `sequences` here. ( read about it in FactoryGirl )